### ISSUE TITLE
Pymor Issue 

### ISSUE THREAD

**Issue by iFonzo1337** [coats1337/Discordapp-TokenGrabber#10]
Why does it say pymor is not recognized as an internal or external command when i downloaded it and put main.py on my desktop i Don't know if im slow or not but please help!!

**Comment by coats1337** [comment]
did u `pip install pyarmor` ?

**Comment by iFonzo1337** [comment]
Yes  I did and it still don't work let me try again!

**Comment by coats1337** [comment]
any chance you have multiple python versions?

**Comment by iFonzo1337** [comment]
yee i had another version of python downloaded. I uninstalled it and redid the setup and it still says pyarmor is not recognized

**Comment by coats1337** [comment]
windows + r and type `python`, which version does it open?

**Comment by iFonzo1337** [comment]
Im running 3.8.3

**Comment by coats1337** [comment]
when you redid the setup, did u make sure to `pip install pyarmor` again?

**Comment by coats1337** [comment]
eh weird, i'll leave this issue open, i can't really help.

**Comment by Cenatel** [comment]
Maybe the python Scripts folder isn't added to PATH ?

**Comment by lolking04** [comment]
Same problem

**Comment by 4nekura** [comment]
I found the fix. C:\Users\YOUR_USERNAME\AppData\Local\Packages You gotta delete all the Python Packages in this folder. Then reinstall and it should be fixed.

**Comment by iFonzo1337** [comment]
> I found the fix. C:\Users\YOUR_USERNAME\AppData\Local\Packages You gotta delete all the Python Packages in this folder. Then reinstall and it should be fixed.

That's how you fix the pyarmor issue

**Comment by coats1337** [comment]
Readme.MD, it's been decided ill leave this discontinued while i work on a new version.
